lib/src/utils/commands_extension.dart

```diff
@@ -91,8 +98,12 @@ extension CommandsClientExtension on Client {
 
   /// Register all default commands
   void registerDefaultCommands() {
-    addCommand('send', (CommandArgs args) async {
-      return await args.room.sendTextEvent(
+    addCommand('send', (args, stdout) async {
```

Comment:
we could add the `eventId` in the stdout here?

Reply:
I'd object this since sending an Event is the default "action" on the `Room.send`. The event ID always used to be returned as result of the method, not as command output. I'd rather see the standard output StrinBuffer as a complimentary concept for *additional* information which is exactly *not* the default behavior (aka sending an event).

Reply:
Consider the following flow :

As a user, I enter something into a message text field of my matrix client. I send the contents via the `Room.send*` methods.

The matrix client underneath now does not know at first if I just sent a regular message or if a command was executed. As the matrix client, I will know about this by checking the stdout. If there is data in the stdout, I consider the result as command result and show e.g. a dialog or a snack bar with a details button where I can see the stdout.

If the stdout is empty and I simply got back an event id as return value, as a matrix client, I simply know the user just sent a message.

If we'd by default add the event id to the stdout, the entire flow of determination whether a command with special result was executed or just a simple event was sent stops working.

Reply:
I'm just trying to understand the use case for stdout once again, pls correct me if I'm wrong.

I use `Room.send` with a text while I'm unsure if it's actually a command or just a text that needs to be sent. So, what I do is I create a `stdout` object that will return me something (can be any meaningful data) when it's **a command that does something other than sending an event**.

If that's the case, as the user of the SDK I want 2 things to be done:
1. Dartdoc comments that clearly explains this context wherever `stdout` is mentioned.
2. An example usage of this scenario in both README and dartdoc comment above `Room.send`.

If I didn't understand it correctly, please tell me.